discord/colour.py

```diff
@@ -119,17 +119,25 @@ def default(cls):
         return cls(0)
 
     @classmethod
-    def random(cls):
+    def random(cls, seed=None):
         """A factory method that returns a :class:`Colour` with a random hue.
 
         .. note::
 
             The random algorithm works by choosing a colour with a random hue but
             with maxed out saturation and value.
 
+        Parameters
+        ------------
+        seed: Optional[Union[:class:`int`, :class:`str`, :class:`float`, :class:`bytes`, :class:`bytearray`]]
+            The seed to initialize the colour with.
+
```

Comment:
Parameters should be below the 1.6 versionadded marker

Should also have a 1.7 versionadded under the seed parameter

Reply:
No? Should it not be a function-wide `versionchanged`?

Reply:
The [documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionchanged) implicitly states that `.. versionchanged::` should be used here, but I've seen both used in this context throughout the library - though I do also recall a consistency PR changing this.